backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/components/CassandraTable.java

```diff
@@ -19,9 +19,27 @@
 
 package org.apache.james.backends.cassandra.components;
 
+import java.util.Objects;
+
+import com.datastax.driver.core.KeyspaceMetadata;
+import com.datastax.driver.core.Session;
 import com.datastax.driver.core.Statement;
+import com.google.common.base.MoreObjects;
 
 public class CassandraTable {
+    public enum InitializationStatus {
```

Comment:
I still can't find out why `CassandraTable.InitializationStatus != CassandraType.InitializationStatus`.

Maybe I'm 'perfectionist' but they share the exact same internals, represent the exact same thing, and you even merge the exact same entity!


I don't see which benefits being 'type specific' brings. could you develop?